setup.cfg

```diff
@@ -36,12 +36,12 @@ install_requires =
     tornado>=6.1.0
     jupyter_core
     jupyterlab_server~=2.10
-    jupyter_server~=1.4
+    jupyter_server~=1.8
     nbclassic~=0.2
-    jinja2>=2.1
+    jinja2>=3.0.3
```

Comment:
```suggestion
    jinja2>=2.1
```

Reply:
We'll have to pin `markupsafe` then I believe